src/app/hearing/components/view-hearing/view-hearing.component.html

```diff
@@ -0,0 +1,122 @@
+<mat-card *ngIf="hearing">
+  <h2>
+    <span class="case-number">{{hearing.caseNumber}}</span>
+    {{hearing.caseTitle}}
+  </h2>
+  <div class="grid">
+    <div class="tile">
+      <strong>Case type</strong><br>
+      {{hearing.caseType}}
+    </div>
+
+    <div class="tile">
+      <strong>Hearing Type</strong><br>
+      {{hearing.hearingType}}
+    </div>
+
+    <div class="tile">
+      <strong>Priority</strong><br>
+      {{hearing.priority}}
+    </div>
+
+    <div class="tile">
+      <strong>Estimated duration</strong><br>
+      {{formatDuration(hearing.duration)}}
+    </div>
+
+    <div class="tile">
+      <strong>List between</strong><br>
+      {{getListBetween()}}
+    </div>
+
+    <div class="tile">
+    </div>
+
+    <div class="tile">
+      <strong>Reserved for</strong><br>
+      {{hearing.reservedToJudge}}
+    </div>
+
+    <div class="tile">
+      <strong>Communication facilitator</strong><br>
+      {{hearing.communicationFacilitator}}
+    </div>
+
+    <div class="tile">
+    </div>
+
+    <div class="tile">
+      <strong>Special requirements</strong><br>
+      {{hearing.specialRequirements}}
+    </div>
+
+    <div class="tile">
+      <strong>Facility requirements</strong><br>
+      {{hearing.facilityRequirements}}
+    </div>
+
+    <div class="clear"></div>
+  </div>
+  <mat-tab-group>
+
+    <mat-tab label="Notes">
+      <mat-list class="hearing-notes">
+        <mat-list-item *ngFor="let note of hearing.notes" class="note">
+          <div class="content">
```

Comment:
Isn't there already a component which could be reused for displaying note?

Reply:
+1